aim/api/infra.py

```diff
@@ -154,3 +155,43 @@ def is_available(self, context):
             LOG.debug("APIC %s is not available. Last update time was %s" %
                       (self.apic_host, self.last_update_timestamp))
             return False
+
+
+class NestedParameter(resource.ResourceBase):
```

Comment:
We probably should put a revisit that we need to deprecate these resources once there's a proper fix in the OpenShift Installer.

Reply:
sure will do.